.changelog/118.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+tfsdk: Don't attempt validation on a Null-valued `SingleNestedAttribute`
```

Comment:
```suggestion
tfsdk: Don't attempt validation on the nested attributes of a null or unknown `SingleNestedAttribute`
```

I think the validators on the attribute itself will still run, right? This is just for the nested attributes of that attribute?

Reply:
Correct.